ledger-core/v2/network/pulsenetwork/distributor.go

```diff
@@ -67,13 +76,22 @@ func NewDistributor(conf configuration.PulseDistributor) (insolar.PulseDistribut
 }
 
 func (d *distributor) Init(ctx context.Context) error {
-	handler := hostnetwork.NewStreamHandler(func(context.Context, *packet.ReceivedPacket) {}, d.responseHandler)
-
 	var err error
-	d.transport, err = d.Factory.CreateStreamTransport(handler)
+	d.transport, err = d.Factory.CreateDatagramTransport(handlerThatPanics{})
```

Comment:
I believe this is wrong according to principle of "defensive external APIs". This code shouldn't panic, but log unexpected incoming datagrams.